src/main/java/org/opentripplanner/street/model/edge/StreetEdgeReluctanceCalculator.java

```diff
@@ -16,10 +16,14 @@ static double computeReluctance(
     RoutingPreferences pref,
     TraverseMode traverseMode,
     boolean walkingBike,
-    boolean edgeIsStairs
+    boolean edgeIsStairs,
+    boolean edgeIsEscalator
   ) {
     if (edgeIsStairs) {
       return pref.walk().stairsReluctance();
+    } else if (edgeIsEscalator) {
+      if (traverseMode == TraverseMode.BICYCLE) return Double.POSITIVE_INFINITY;
```

Comment:
I think we shouldn't use really high cost but instead prevent "bicycle walking" and cycling in escalators completely.